### PR TITLE
[PR] Migrates Toast component

### PR DESCRIPTION
**Pull request by denistsoi**
Closes https://github.com/thenewboston-developers/ui-js/issues/15

(Doesn't include storybook for QR component, will do that in another PR).
acc: baa336bca22c7590256a86c002e4dbe6c1f1c1f6c0a0fd3f0681eecb2cad0787

### CODE REVIEW

**Comment by angle943** [comment]
can you add the storybook too?

**Comment by denistsoi** [comment]
Storybook evidence

Screenshot 2021-04-13 at 12 39 37 AM

File: src/components/Toast/Toast.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  border-left-width: 3px;
+  display: flex;
+  padding: 12px;
+
+  &--success {
+    border-left-color: var(--color-tertiary);
+  }
+
+  &--warning {
+    border-left-color: var(--color-danger);
```
Comment (thesanjeevsharma) [inline src/components/Toast/Toast.scss:12]: @denistsoi You forgot to add this color in styles/colors.css

File: src/components/Toast/Toast.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import { Meta, Story } from '@storybook/react';
+
+import Toast, { ToastProps } from './index';
+import '../../styles/core.css';
+import '../../styles/colors.css';
+import './Toast.scss';
+
+const meta: Meta = {
+  title: 'Toast',
+  component: Toast as any,
```
Comment (thesanjeevsharma) [inline src/components/Toast/Toast.stories.tsx:11]: Why is `any` required here?

File: src/components/Toast/index.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import './Toast.scss';
+
+export type ToastType = 'success' | 'warning';
+
+export interface ToastProps {
+  className?: string;
+  type: ToastType;
+}
+
+const Toast: FC<ToastProps> = ({ children, className, type = 'warning' }) => {
```
Comment (thesanjeevsharma) [inline src/components/Toast/index.tsx:15]: Can you add description comments here? Check Inputs/Button component.

**Review by jamessspanggg — APPROVED** [review]

**Comment by denistsoi** [comment]
@jamessspanggg - same here...

**Comment by jamessspanggg** [comment]
@angle943

**Comment by angle943** [comment]
PAID

**Pull request merged**